src/js/components/search/filters/timePeriod/TimePeriod.jsx

```diff
@@ -94,7 +129,12 @@ export default class TimePeriod extends React.Component {
         }
 
         if (this.state.shownFilter === 'fy') {
```

Comment:
Why is this in here twice? Almost identical code is in lines 117-124. What is the case that this `if` is handling that isn't handled there? If this logical structure is necessary, is there a way to make this less repetitive?